src/openms/include/OpenMS/APPLICATIONS/OpenSwathBase.h

```diff
@@ -377,13 +376,13 @@ class TOPPOpenSwathBase :
       String model_type = irt_detection_param.getValue("alignmentMethod").toString();
       trafo_rtnorm.fitModel(model_type, model_params);
     }
-    else if (!irt_tr_file.empty())
+    else if (!irt_transitions.getTransitions().empty())
     {
       // Loading iRT file
       std::cout << "Will load iRT transitions and try to find iRT peptides" << std::endl;
-      FileTypes::Type tr_type = FileHandler::getType(irt_tr_file);
-      Param tsv_reader_param = TransitionTSVFile().getDefaults();
-      OpenSwath::LightTargetedExperiment irt_transitions = loadTransitionList(tr_type, irt_tr_file, tsv_reader_param);
+//      FileTypes::Type tr_type = FileHandler::getType(irt_tr_file);
+//      Param tsv_reader_param = TransitionTSVFile().getDefaults();
+//      OpenSwath::LightTargetedExperiment irt_transitions = loadTransitionList(tr_type, irt_tr_file, tsv_reader_param);
```

Comment:
?

Reply:
Oh right, this was the original code for loading the iRTs, which I commented out in case I needed it. But I've removed it now in the recent commit.